scopes/generator/generator/component-generator.ts

```diff
@@ -82,6 +82,7 @@ export class ComponentGenerator {
       componentName: componentId.fullName,
       defaultScope: this.options.scope,
     });
+    await this.workspace.triggerOnComponentAdd(componentId);
```

Comment:
@davidfirst what do you think about calling this event here?
will it do any harm?
do you think it will be better to split it into 2 different events?
1. for add/create when watching is running
2. for add/create when watching is not running

Reply:
I don't think it's needed, unless this is running during watch.
Otherwise, you will only get the compilation from this trigger, which I would prefer to call explicitly

Reply:
What is not needed?
Calling the event or split it into 2 events?
What we need is to make sure when root comps is enabled to run bit install when component is added (both if watch is running or not) this is required in order to create the inner package name of the component which is done by the package manager. Without it, the dists won't be written into the inner dir.

Reply:
> What is not needed?
this line `await this.workspace.triggerOnComponentAdd(componentId);`

> What we need is to make sure when root comps is enabled to run bit install
maybe it'd be best to call the installer directly? if the watch is running in the background, the line above will be running regardless.

Reply:
Added a new event: "onMultipleComponentsAdd"